lib/inc/SaiInterface.h

```diff
@@ -203,6 +203,30 @@ namespace sairedis
                     _In_ uint32_t attr_count,
                     _Inout_ sai_attribute_t *attr_list);
 
+        public: // bulk QUAD meta key
+
+            virtual sai_status_t bulkCreate(
+                    _Inout_ sai_object_meta_key_t* metaKey,
+                    _In_ sai_object_id_t switch_id,
+                    _In_ uint32_t object_count,
+                    _In_ uint32_t *attr_count,
+                    _In_ const sai_attribute_t **attr_list,
```

Comment:
also on pointers we prefer to use * on the like "sai_attribute_t** attr_list"

Reply:
we have **attr_list definition in Meta for bulkCreate (src/sonic-sairedis/meta/Meta.cpp:2563)
```
sai_status_t Meta::bulkCreate(
        _In_ sai_object_type_t object_type,
        _In_ sai_object_id_t switchId,
        _In_ uint32_t object_count,
        _In_ const uint32_t *attr_count,
        _In_ const sai_attribute_t **attr_list,
        _In_ sai_bulk_op_error_mode_t mode,
        _Out_ sai_object_id_t *object_id,
        _Out_ sai_status_t *object_statuses)
{
```

Reply:
ok